mlflow/spark.py

```diff
@@ -728,7 +734,13 @@ def predict(self, pandas_df):
         :param pandas_df: pandas DataFrame containing input data.
         :return: List with model predictions.
         """
+        from pyspark.ml import PipelineModel
+
         spark_df = self.spark.createDataFrame(pandas_df)
+        if isinstance(self.spark_model, PipelineModel):
+            # make sure predict work by default for Transformers
+            if self.spark_model.stages[-1].hasParam("outputCol"):
```

Comment:
```suggestion
        if isinstance(self.spark_model, PipelineModel) and self.spark_model.stages[-1].hasParam("outputCol"):
            # make sure predict work by default for Transformers
```

can we merge the conditions?